src/License/Repositories/LicenseRepository.php

```diff
@@ -104,6 +110,58 @@ public function getPlatformFeePercentage(): float
         return $this->getStoredPlatformFeePercentage();
     }
 
+    /**
+     * Check if licenses are in the grace period.
+     *
+     * @unreleased
+     */
+    private function isLicenseInGracePeriod(): bool
+    {
+        $currentTime = current_time('timestamp', true);
+        
+        $lastActiveDate = get_option(LicenseOptionKeys::LAST_ACTIVE_LICENSE_DATE);
```

Comment:
This can be replaces with the new method:

```suggestion
        $lastActiveDate = $this->getLastActiveLicenseDate();
```